src/breadcrumbs/index.cjsx

```diff
@@ -47,39 +48,47 @@ Breadcrumbs = React.createClass
     task: tasks.get(taskId)
     moduleInfo: tasks.getModuleInfo(taskId)
 
-  render: ->
-    {task, moduleInfo} = @state
-    {currentStep, canReview} = @props
-
-    return null if _.isEmpty(task.steps)
-
-    crumbs = _.map(task.steps, (crumbStep, index) ->
-      crumb =
-        key: index
-        data: crumbStep
-        crumb: true
-        type: 'step'
-    )
+  makeCrumbEnd: (type, enabled) ->
+    {moduleInfo} = @state
 
     reviewEnd =
       type: 'end'
-      key: crumbs.length
       data:
-        id: ''
+        id: "#{type}"
         title: moduleInfo.title
-      disabled: not canReview
+      label: type
+      disabled: not enabled
+
+  render: ->
+    {task, moduleInfo} = @state
+    {currentStep, canReview, shouldContinue} = @props
+    return null if _.isEmpty(task.steps)
+
+    crumbs = _.map task.steps, (crumbStep, index) ->
+      data: crumbStep
+      crumb: true
+      type: 'step'
+
+    reviewEnd = @makeCrumbEnd('summary', canReview)
+    bookEnd = @makeCrumbEnd('continue', shouldContinue)
 
     crumbs.push(reviewEnd)
+    crumbs.push(bookEnd)
+
+    breadcrumbs = _.map crumbs, (crumb, index) =>
+      {disabled} = crumb
+      classes = classnames({disabled})
+      crumb.key = index
 
-    breadcrumbs = _.map(crumbs, (crumb) =>
       <BreadcrumbDynamic
-        className={'disabled' if crumb.disabled}
+        className={classes}
+        data-type={crumb.label}
```

Comment:
Does `crumb.label` contain spaces or upper-case? If so, maybe this should be `data-label` instead?